config/default.yml

```diff
@@ -106,3 +106,8 @@ site_name:
     fi: 'Pääkaupunkiseudun palvelukartta'
     sv: 'Servicekarta'
     en: 'Service Map'
+
+# settings for general feedback form
+open311_service_code: 4a872cbf-aefa-e711-80f5-001dd8b71c08
+open311_api_key: ''
+send_only_general_feedback: true
```

Comment:
I don't really see a huge need for this since there is no action taken if this is `false`. I would rather just do a check for `open311_service_code` and `open311_api_key` and if both of them as set, then set then in the `json` object for instance.